src/pages/Homepage.js

```diff
@@ -29,48 +40,53 @@ function Home() {
     }
   };
 
+  //For loading income and expenses automatically
   useEffect(() => {
     getIncome();
     dispatch(handleFetchExpenses());
   }, []);
 
-  //GET-EXPENSE
-  // const getExpense = async () => {
-  //   try {
-  //     const res = await axios.get(`${apiUrl}/api/expense`);
-  //     setExpense(res.data);
-  //     setFetchError(null);
-  //   } catch (err) {
-  //     setFetchError(err.message);
-  //   }
-  // };
-
-  // useEffect(() => {
-  // }, []);
-
+  //For logging out
   function logout() {
     localStorage.removeItem("token");
     navigate("/");
   }
 
-  // function expenseInformation() {
-  //   <ExpenseInformation
-  //     register={register}
-  //     handleSubmit={handleSubmit}
-  //     dispatch={dispatch}
-  //   />;
-  // }
-
-  // const incomeData = {
-  //   title: income.title,
-  //   amount: income.amount,
-  //   date: income.date,
-  //   created_at: income.created_at,
-  // };
+  //For adding more expenses
+  function expenseInformation() {
+    navigate("/expenseinput");
```

Comment:
change path to /expense-input or /expense/input

Reply:
Done!